### DISCUSSION TITLE
BlackSea Visualization

### DISCUSSION THREAD

**Discussion by alioacar** [pyvista/pyvista#6774] [Show and tell]
### Describe something you've made

Hi everyone!

I wanted to share a visualization I created using PyVista for the Black Sea bathymetry and surrounding terrain, and say thank you for this amazing library! This visualization is part of my work towards displaying coupled hydrodynamics-biogeochemistry modeling results in 3D.
The ultimate goal is to use this bathymetry as a base layer to visualize oceanographic parameters and biogeochemical processes in the Black Sea. Thanks again for maintaining such a powerful and user-friendly visualization tool!

### Pseudocode or Screenshots

![black_sea](https://github.com/user-attachments/assets/99ebb0b8-d13a-4534-b35a-6baaef07de85)

**Comment by banesullivan** [comment]
This is so, so excellent! Thank you for posting here to share this work! If you're feeling up for it, I'm curious to know your answers to some of the questions I ask in https://github.com/pyvista/pyvista/discussions/2133

> I'm hoping we can use this discussion topic to gain insight on not only people's overall experience (for better or worse) using PyVista, but also understand where people are coming from, how PyVista impacts their work, and where the community would like PyVista to go.
> 
> Some questions I have in mind that would be awesome if you could answer or expand on are (please suggest other questions/topics as well!):
> 
> - what is your domain/background?
> - what is your familiarity with Python and the Python Data Science Stack?
> - what is your experience level with 3D visualization?
> - have you used other 3D visualization software in Python or otherwise (open source or otherwise)?
> - what was your experience between these software?
> - what are your primary visualization tools?
> - where would you like to see PyVista go?
